libs/test_positivity.py

```diff
@@ -21,35 +22,83 @@
 _log = structlog.get_logger()
 
 
-@dataclasses.dataclass
-class Method:
+class Method(ABC):
```

Comment:
I refactored this to have a `Method` base class and a `DivisionMethod` and `PassThruMethod` derived classes.  python OO is new to me so holler if this doesn't seem right.

Reply:
Ha, I made a very similar change in a branch while fiddling with this.